Core/src/main/java/org/tribuo/Example.java

```diff
@@ -307,6 +307,8 @@ public void transform(TransformerMap transformerMap) {
 
     /**
      * Converts all implicit zeros into explicit zeros based on the supplied feature map.
```

Comment:
This docstring should refer users to implementation specific docstrings that may impose restrictions (ie. must be sorted) on its input.

Reply:
I added a note saying this method is optional and implementations may have additional restrictions.